src/platform/unittest/TlsTest.cpp

```diff
@@ -29,7 +29,9 @@ struct TlsTest : public ::testing::TestWithParam<bool>
     CXPLAT_SEC_CONFIG* ClientSecConfigCustomCertValidation {nullptr};
     CXPLAT_SEC_CONFIG* ClientSecConfigExtraCertValidation {nullptr};
     CXPLAT_SEC_CONFIG* ClientSecConfigNoCertValidation {nullptr};
+    CXPLAT_SEC_CONFIG* ClientSecConfigClientCert {nullptr};
```

Comment:
Rename to indicate there is no cert validation:
```suggestion
    CXPLAT_SEC_CONFIG* ClientSecConfigClientCertNoCertValidation {nullptr};
```